docs/data/material/customization/containerQueries/BasicContainerQueries.js

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Chip from '@mui/material/Chip';
+import Typography from '@mui/material/Typography';
+import { styled } from '@mui/material/styles';
+
+const DynamicCard = styled(Card)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  [theme.containerQueries.up(350)]: {
+    flexDirection: 'row',
+  },
+}));
+
+const Image = styled('img')(({ theme }) => ({
+  alignSelf: 'stretch',
+  aspectRatio: '16 / 9',
+  objectFit: 'cover',
+  width: '100%',
+  maxHeight: 160,
+  transition: '0.4s',
+  [theme.containerQueries.up(350)]: {
+    maxWidth: '36%',
+    maxHeight: 'initial',
+  },
+  [theme.containerQueries.up(500)]: {
+    maxWidth: 240,
+  },
+}));
+
+const Content = styled(CardContent)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(1),
+  padding: theme.spacing(2),
+  flex: 'auto',
+  transition: 'padding 0.4s',
+  [theme.containerQueries.up(500)]: {
+    padding: theme.spacing(3),
+  },
+}));
+
+const Line = styled('div')(({ theme }) => ({
```

Comment:
We could take advantage of the multi-tab demo container here and have everything that's a utility for the demo, like this Line component, in the second tab, thus only showing in the first tab code that's relevant to demonstrate the use of `theme.containerQueries`.